src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx

```diff
@@ -58,15 +58,13 @@ The Kubernetes page in APM offers the following information about your applicati
       src={kubernetesHeader}
     />
      
-    If you have an application that spans **multiple clusters**, you can select a specific Kubernetes cluster from the **Cluster** drop-down list, and isolate the deployment to that targeted cluster.
+    If you have a multi-cluster app (an app that spans multiple clusters), when you go to the **Kubernetes** page, a summary of your clusters is displayed. To isolate a specific Kubernetes cluster, select it from the **Cluster** drop-down list. To return to a view of all clusters, click **All clusters**.
```

Comment:
I'd do:

If you have a multi-cluster app, that is, an app that spans multiple clusters, you can check a specific cluster. To do so, select it from the **Cluster** drop-down list. Then, click **All clusters** to return to a view of all clusters.